Created toolbar and headline section

ogoigbe12 opened the pull request:
started with adding linters to the project and also added profile.html and also style.css and both were updated with the task I was asked to perform also added a README.md file which was also updated with professional format all this was done in the created_toolbar-and-headline-section branch not the master branch

ogoigbe12 committed:
fest: added linters config
- add .gitignore
- add .hintrc
- add  .stylelintrc.json
- add package.json
- add package-lock.json

ogoigbe12 committed:
add fest:
- update style.css
-  add micro.png
- update index.html
- update README.md

ogoigbe12 committed:
change index.html to profile.html

ogoigbe12 committed:
update to style.css

ogoigbe12 committed:
update style.css

ogoigbe12 committed:
update style.css

Felix45 commented on porfile.html:
```diff
@@ -0,0 +1,63 @@
+    <img src="img/Rectangle 4.svg" alt="Rectangle 4" id="Rectangle4">
+    <img src="img/Ellipse 3.svg" alt="Ellipse 3" id="Ellipse3">
+
+</div>
```
- [ Required ] It's always advisable that when an image is used for decoration on a webpage it should be included as a `background-image`, therefore I would like to advise that you remove this `div` from your design.

Felix45 commented on porfile.html:
```diff
@@ -0,0 +1,63 @@
+                <img src="img/Rectangle 5.svg" alt="Rectangle 5" id="Rectangle5">
+                 <img src="img/Ellipse 4.svg" alt="Ellipse 4" id="Ellipse4">  
+            </div>
+        </section>
```
- [ Required ] It's always advisable that when an image is used for decoration on a webpage it should be included as a `background-image`, therefore I would like to advise that you remove this `section` from your design.

Felix45 commented on porfile.html:
```diff
@@ -0,0 +1,63 @@
+          
+    <section id="headline">
+        <h1 class="title">I'm Samuel Ogoigbe</h1>
+        <h1 class="title">Glad to see you!</h1>
```
- [ Required ] A `h1` tag is usually used to mark the most important section of a webpage, it's therefore advisable that a webpage ought to have only one `h1` element. I would like to suggest that you change the second `h1` tag to a `h2` tag for your webpage to be semantically correct.

Felix45 commented on porfile.html:
```diff
@@ -0,0 +1,63 @@
+
+</div>
+          
+    <section id="headline">
```
- [ Required ] Please consider giving this section a `background-image` that is consistent with the `Figma` design, currently, the `background-image`, has some variations compared to the Figma design. I suggest using a `single` background image that has all the `shapes` present in the `Figma` design as a background image for this section. Please see the screenshot below for more details on this.

## Figma design
<img src="https://user-images.githubusercontent.com/16392046/219791833-3c821b6d-83c2-4836-9328-71b3b668bd79.png" width="375px" />

## Your design
<img src="https://user-images.githubusercontent.com/16392046/219792255-9779a519-bffe-4115-823c-4d38d0752a6d.png" />

## Background image to use
<img src="https://user-images.githubusercontent.com/16392046/219795053-8efd0698-a13a-4529-9a7d-f2bcc4bf9074.jpg" width="375px" />

Felix45 commented on porfile.html:
```diff
@@ -0,0 +1,63 @@
+    <section id="headline">
+        <h1 class="title">I'm Samuel Ogoigbe</h1>
+        <h1 class="title">Glad to see you!</h1>
+        <p class="description">
+            I'm a software developer! I can help you build a product , feature or website Look through some of my work
+            and
+            experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+        </p>
+        <p class="contact-link">LET'S CONNECT</p>
+        <div class="svg-img">
+            
+              <a href="#"><img src="Assets/twitter.svg" alt="twitter link"></a>
+              <a href="#"><img src="Assets/Linkedin icon.svg" alt="linked link"></a>
+              <a href="#"><img src="Assets/medium.svg" alt="medium"></a> 
+              <a href="#"><img src="Assets/github.svg" alt="github link"></a>
+              <a href="#"><img src="Assets/angellist.svg" alt="doit"></a>
+            
+         </div>
```
- [ Required ] I think it would be better if the `margins` and `paddings` of this part of your webpage are consistent with the `Figma` design, currently, the elements on your page appear to be too `close` to each other than they are in the `Figma` design. Please see the screenshot below for more details on this and rectify it.

## Figma design
<img src="https://user-images.githubusercontent.com/16392046/219794236-a53fd0ab-2bd9-466d-8763-c1dd7aa2540e.png" width="375px" />

## Your design
<img src="https://user-images.githubusercontent.com/16392046/219793748-3669b52d-1e12-42c9-a9de-3c2d203fb0cf.png" width="375px" />

Felix45 reviewed (changes requested):
Hi @ogoigbe12 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

<img src="https://camo.githubusercontent.com/59fa9d95c18dbd5a8e47490a9e4b48393f569685410f3c78d4d83a9764fa1392/68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f74385950663341434b656f754c7a454976722f67697068792e676966" />

## Highlights :heavy_check_mark: 
- [ ] No linter errors :white_check_mark: 
- [ ] GitHub flow is used correctly :white_check_mark: 


## Required Changes ♻️

- [ Required ] Please consider adding a `License` to your project directory, you can use the `MIT` License for this. Once the `License` is in your project directory you can add a link to the License on the `readme` file. A License serves to inform someone who would like to use your code, how the project is Licensed.

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

ogoigbe12 committed:
add fest:
- image icon
- update porfile.html
- update style.css

ogoigbe12 committed:
update style.css

ogoigbe12 committed:
update style.css

ogoigbe12 committed:
update style.css

ogoigbe12 committed:
update style.css

ogoigbe12 committed:
update style.css

ogoigbe12 committed:
update style.css

ogoigbe12 committed:
update style.css

ogoigbe12 committed:
update style.css

ogoigbe12 committed:
update style.css

bobb-Rob commented on porfile.html:
```diff
@@ -0,0 +1,61 @@
+            <button id="hamburger"><img width="16" src="Assets/icons8-hamburger-32 (1).png" alt="Menu"></button>
+        </nav>
+    </header>
+    <section class="headline">
+        <div class="content">
+            <h1 class="title">
+                I'm Samuel ogoigbe
+                <br>
+                Glad to see you!
+            </h1>
+            <p class="description">
+                I'm a software developer! I can help you build a product , feature or website Look through some of my
+                work
+                and
+                experience! If you like what you see and have a project you need coded, don't hestiate to contact me.
+            </p>
+            <p class="connect-link">
+                LET'S CONNECT
```
- [ ] As observed in the Figma design template, the headline section has a background image. Please add the proper background image (decorations are always background images, not `img` tags). Usually, if an image is not part of the content (aesthetic reasons) use CSS background-image. [Check this link](https://stackoverflow.com/questions/492809/when-to-use-img-vs-css-background-image) for more info.
- [ ] The left bottom border-radius in your design is not similar to that of the Figma template. Kindly fix this.

Your Design | Figma Design
---- | ----
![image](https://user-images.githubusercontent.com/84156714/220061704-f1b56199-f842-4dc5-805d-d79fab0e11f7.png) | ![image](https://user-images.githubusercontent.com/84156714/220062560-be00df27-f4a8-41e3-8204-1dadc1307b52.png)

bobb-Rob commented on porfile.html:
```diff
@@ -0,0 +1,61 @@
+<body>
+    <header>
+        <nav class="navbar">
+            <a href="index.html" class="brand">My Logo</a>
```
- [ ] Your logo is not supposed to be inside the `nav` element. Kindly remove it and place it directly inside the `header`. Also, change the logo text from `my Logo` to `Your name`.

bobb-Rob commented on README.md:
```diff
@@ -0,0 +1,96 @@
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Hello Microverse] <a name="about-project"></a>
+
+**[Hello Microverse]** is a git and github training project.
```
- [ ] This README title is not reflecting this project. Kindly edit your README to suit this project.

bobb-Rob commented on README.md:
```diff
@@ -0,0 +1,96 @@
+
+**[Hello Microverse]** is a git and github training project.
+
+## 🛠 Built With <a name="built-with"></a>
```
- [ ] [OPTIONAL] I notice that your commit messages are not very professional. Please read through [this article](https://www.freecodecamp.org/news/writing-good-commit-messages-a-practical-guide/) and I believe you will learn best practices.

👇🏾 
![image](https://user-images.githubusercontent.com/84156714/220065039-22821b1f-e62b-4ce7-9b78-3fdd2fafa1e0.png)

bobb-Rob reviewed (changes requested):
## STATUS: CHANGES REQUIRED ♻️
Hi @ogoigbe12 , 

Good job so far! 👏👏👏 
There are some issues that you still need to work on to go to the next project but you are almost there!  😃 
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>
![](https://camo.githubusercontent.com/34482800e1b92cb79ddca06edb4051edc407cb190f896e1e949e596aa9073851/68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f556461535559436c723932636a57706556782f67697068792e676966)

## Highlights
✅ No Linter error
✅ Correct use of GitHub flow
✅ Very descriptive Pull Request(PR)
✅ Neat Commit messages

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

levy002 commented on README.md:
```diff
@@ -0,0 +1,96 @@
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
```
- [OPTIONAL] Kindly make sure to create the `LICENCE` and add it to this repo and make sure this link directs user the created `LICENCE` file.

levy002 reviewed (approved):
## Hello @ogoigbe12 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

![](https://media.giphy.com/media/jJQC2puVZpTMO4vUs0/giphy.gif)

## Status: APPROVED :fire:  :fire: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

ogoigbe12 committed:
Add fest:
- Deleted all elipse and rectangle image
- update profile.html
- update style.css

ogoigbe12 committed:
update style.css

ogoigbe12 committed:
add fest:
-update README.md
-deleted micro.png
-add sammie.jpg

ogoigbe12 committed:
add fest:
-update profile.html
-update README.md
-update style.css
-delete background.svg
-add background.img
